### ISSUE TITLE
Install issues

### ISSUE THREAD

**Issue by billbrod** [cduvallet/microbiomeHD#1]
I'm having a couple install issues when attempting to install the requirements on NYU's HPC cluster and using virtualenv to make sure it's a clean install. I'm using `pip3 install -r requirements.txt` to try and get everything.

- you should probably mention somewhere that it requires python 3. I'm one of those luddites who still uses 2 by default (version 0.5.1 one scikit-bio is only python 3 compatible)

- feather requires Cython but when installing with pip for some reason it doesn't get installed first (raises `ModuleNotFoundError: No module named 'Cython'`). running `pip3 install Cython` seems to fix this.

- I think feather also requires numpy and doesn't install it properly (`ModuleNotFoundError: No module named 'numpy'`). Again, running `pip3 install numpy` seems to fix it.

**Comment by cduvallet** [comment]
I am actually *also* one of those luddites, and this is all written in python2! I'll make sure to specify that in the README.

When using `pip install -r requirements.txt` (i.e. with python 2), I got the scikit-bio error you mentioned to me yesterday, which seems to be this open sciki-kit bio [issue](https://github.com/biocore/scikit-bio/issues/1433). I've updated the `requirements.txt` file with the right version specified (commit 4751b5589121a390c8e4a1f5e30039d0fa6a1326).

Can you try again and let me know if it works now? 

Also, I'm still not 100% sure how virtual environments work, but I also ran my install using a virtual environment and didn't get an error installing feather. So either it found an already installed global version of it, or their pip bug got fixed?

**Comment by billbrod** [comment]
Alright, with that update, I was able to install everything (with python 2).

I still had to run `pip install feather_format==0.3.1` beforehand, so I don't know why it worked for you and not for me (I'm not entirely clear on how virtual environments either).

I also ran into [this](https://github.com/pypa/pip/issues/3532) issue, and so the command I ended up using was `pip install --no-cache-dir -r requirements.txt`, but I don't think that's an issue with anything on your end.
